### PR TITLE
feat-input

### PR DESCRIPTION
**Pull request by abhij1607**
image

### CODE REVIEW

**Review by MaruthiPothuganti — APPROVED** [review]
Looks good to merge

File: components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.input {
+  display: inline-block;
+  vertical-align: middle;
+  border: 1px solid transparent;
+  padding: 5px 10px;
+  font-size: 1rem;
+  border-radius: 2rem;
+}
+.input:focus {
+  outline: none;
```
Comment (AnuragSahu11) [inline components/Input/input.css]: outline none  is not considered good practice, instead use this  outline: 2px solid transparent; outline-offset: 2px;

**Review by AnuragSahu11 — APPROVED** [review]
Keep up the good work, code looks good you can merge

File: components/Input/input.css (hunk 1 above)
Comment (abhij1607) [reply components/Input/input.css]: sure, I'll make suggested change

**Pull request merged**